must-gather/collection-scripts/gather

```diff
@@ -19,15 +20,17 @@ gather_clusterscoped_resources ${BASE_COLLECTION_PATH} "${SINCE_TIME}"
 gather_noobaa_resources ${BASE_COLLECTION_PATH} "${SINCE_TIME}"
 
 if [ "$(oc get storagecluster -n openshift-storage -o go-template='{{range .items}}{{.spec.externalStorage.enable}}{{"\n"}}{{end}}')" == true ]; then
-    echo "Skipping the ceph collection as External Storage is enabled"
-elif [ -z "$(oc get storagecluster -n openshift-storage -o go-template='{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}')" ]; then
-    echo "Skipping ceph collection as Storage Cluster is not present"
+    echo "Skipping the ceph collection as External Storage is enabled" | tee -a  "${BASE_COLLECTION_PATH}"/gather-debug.log
+elif [ -z "${storageClusterPresent}" ]; then
+    echo "Skipping ceph collection as Storage Cluster is not present" | tee -a  "${BASE_COLLECTION_PATH}"/gather-debug.log
 else
     gather_ceph_resources ${BASE_COLLECTION_PATH} "${SINCE_TIME}"
 fi
 
-# Call post-uninstall.sh
-post-uninstall.sh
+if [ "${storageClusterPresent}" ]; then
+    # Call post-uninstall.sh
+    post-uninstall.sh
+fi
```

Comment:
this will also not delete debug pods. make the required changes in post-uninstall.sh script

Reply:
fixed